docs/details/signal.dox

```diff
@@ -174,12 +187,14 @@ functions can perform a 2D convolution on 3D signal and filters.
 For three dimensional inputs with m, n & p sizes along the 0th, 1st & 2nd dimensions
 respectively, given below are the possible batch operations.
 
-| Input Signal Dimensions   | Filter Dimensions   | Output Dimensions   | Batch Mode                | Description                                                |
-| :-----------------------: | :-----------------: | :-----------------: | :-----------------------: | :------------                                              |
-| [m n p 1]                 | [a b c 1]           | [m n p 1]           | No Batch                  | Output will be a single convolve array                     |
-| [m n p 1]                 | [a b c d]           | [m n p d]           | Filter is Batched         | d filters applied to same input                            |
-| [m n p q]                 | [a b c 1]           | [m n p q]           | Signal is Batched         | 1 filter applied to q inputs                               |
-| [m n p k]                 | [a b c k]           | [m n p k]           | Identical Batches         | k filters applied to k inputs in one-to-one correspondence |
+| Signal Dimensions  | Filter Dimensions  | Output Dimensions  | Batch Mode         | Description                                                |
+| :----------------: | :----------------: | :----------------: | :----------------: |:-----------------------------------------------------------|
+| \dims{m, n, p, 1}  | \dims{a, b, c, 1}  | \dims{m, n, p, 1}  | No Batch           | Output will be a single convolve array                     |
+| \dims{m, n, p, 1}  | \dims{a, b, c, d}  | \dims{m, n, p, d}  | Filter is Batched  | d filters applied to same input                            |
+| \dims{m, n, p, q}  | \dims{a, b, c, 1}  | \dims{m, n, p, q}  | Signal is Batched  | 1 filter applied to q inputs                               |
+| \dims{m, n, p, k}  | \dims{a, b, c, k}  | \dims{m, n, p, k}  | Identical Batches  | k filters applied to k inputs in one-to-one correspondence |
+
+\note For the sake of above tabular illustrations, we assumed \ref af_conv_mode is \ref AF_CONV_DEFAULT.
```

Comment:
nit: Move this before the table

Reply:
I haven't seen notes or * entries before tables or figures in any articles.

Also, having them below the tables gives separation from the other note from general description of convolution functions that is copied before 1D 2D and 3D function documentation.